addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -13,6 +13,16 @@
     flex: 1;
     flex-direction: row;
     flex-wrap: wrap;
+
+    &.o-composer-open {
+        border-bottom: 1px solid;
+    }
```

Comment:
Here border-top will be needed too in the case of "hasBorder" if we add the padding I suggested on the whole chatter.

Reply:
border-top is needed on `o_ChatterTopbar_button` I think, not on the whole bar

Reply:
true, also make sure when you introduce it that it doesn't flicker by pushing everything down on hover